iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/IidmVersion.java

```diff
@@ -83,25 +81,34 @@ public String getXsd(boolean valid) {
         return "iidm_equipment_V" + toString("_") + ".xsd";
     }
 
-    public static IidmVersion fromNamespaceURI(String namespaceURI) {
-        String version = namespaceURI.substring(namespaceURI.lastIndexOf('/') + 1);
-        IidmVersion v = of(version, "_");
-        String namespaceUriV = v.getNamespaceURI();
-        if (!namespaceURI.equals(namespaceUriV)) {
-            if (v.compareTo(V_1_7) >= 0 && namespaceURI.equals(v.getNamespaceURI(false))) {
-                return v;
-            }
-            throw new PowsyblException("Namespace " + namespaceURI + " is not supported. " +
-                    "The namespace for IIDM XML version " + v.toString(".") + " is: " + namespaceUriV + ".");
-        }
-        return v;
-    }
```

Comment:
Why did you remove this method?
It is a public static one, so this is a breaking change. We can of course remove it, but only if needed (for instance, if the method does no longer make sense).